commons/zenoh-codec/src/network/oam.rs

```diff
@@ -21,8 +21,9 @@ use zenoh_protocol::{
     common::{iext, imsg, ZExtBody},
     network::{
         id,
-        oam::{ext, flag, Oam, OamId},
+        oam::{ext, flag, Oam},
     },
+    transport::oam::OamId,
```

Comment:
This change is simply wrong. Revert.

Reply:
`For style consistency, just use super::OamId and then replace the full path expansioncrate::transport::oam::OamId with OamId.`
I just misunderstood you here, sorry ;)
As far as we have `OamId` shred between layers, maybe it is good to move `Qam` definition to separate module and share it between transport and network layer modules?

Reply:
There are 3 protocol levels: `transport`, `network`, `zenoh`. All of those three have an `Oam` message that is semantically different from one and the other and they may evolve in different directions. Let's keep them separate and let's avoid overlap.

Reply:
Ok, I've got it! Thought that Oam is intended to be the same for all levels (some kind of cross-layer extendable functionality)